api/proto/blockchain.proto

```diff
@@ -52,10 +52,10 @@ message BlockContents {
     // Outputs created in this block.
     repeated external.TxOut outputs = 2;
 
-    /// mint-config transactions in this block coupled with data used to validate them.
+    // mint-config transactions in this block coupled with data used to validate them.
```

Comment:
oh I misunderstood the description, I thought this was adding triple slashes to the comments `///`
protos should only need double slash `//`, not sure we should allow clippy to bark on the generated comments, but going to double quotes seems like a good overall cleanup